src/components/ArticleComments.jsx

```diff
@@ -0,0 +1,149 @@
+import { useContext, useEffect, useState } from "react";
+import { userContext } from "../Context/UserProvider";
+import CommentCard from "./CommentCard";
+import axios from "axios";
+import ErrorPopup from "./ErrorPopup";
+import NotificationPopup from "./NotificationWindow";
+
+export default function ArticleComments({ article_id }) {
+    const [commentsStatus, setCommentsStatus] = useState(true);
+    const { user, setUser } = useContext(userContext);
+    const [postCommentErrStatus, setPostCommentErrStatus] = useState(null);
+    const [postLengthErrStatus, setPostLengthErrStatus] = useState(null);
+    const [comment, setComment] = useState("");
+    const [comments, setComments] = useState([]);
+    const [notificationStatus, setNotificationStatus] = useState(null);
+
+    useEffect(() => {
+        axios
+            .get(`https://nc-news-u90o.onrender.com/api/${article_id}/comments`)
+            .then((response) => {
+                return response.data;
+            })
+            .then((data) => {
+                setComments(data.commentsOnArticles);
+                setCommentsStatus(false);
+            });
+    }, []);
+
+    function handleChange(event) {
+        const value = event.target.value;
+        setComment(value);
+    }
+
+    function handleSubmit(event) {
+        event.preventDefault();
+        console.log(comment.length);
+        if (comment.length < 3) {
+            setPostLengthErrStatus(true);
+        } else {
+            const commentData = {
+                body: comment,
+                article_id: article_id,
+                user: user.username,
+            };
+            axios
+                .post(
+                    `https://nc-news-u90o.onrender.com/api/articles/${article_id}/comments`,
+                    commentData
+                )
+                .then((response) => {
+                    console.log(response.data);
+                })
+                .then(() => {
+                    setNotificationStatus(true);
+                    setComments((currComments) => {
+                        const contextRender = {
+                            author: user.username,
+                            body: comment,
+                            votes: 0,
+                        };
+                        return [contextRender, ...currComments];
+                    });
+                })
+                .catch((err) => {
+                    return setPostCommentErrStatus(err);
+                });
+        }
+    }
+
+    function notificationHandler() {
+        setNotificationStatus(false);
+    }
+
+    function postErrHandler() {
+        setPostCommentErrStatus(null);
+    }
+
+    function postLengthErrHandler() {
+        setPostLengthErrStatus(null);
+    }
+
```

Comment:
Adding a notification window is a valid way to let the user know something's wrong, could you change the colour of it so it's easier to see (i.e. not have a black background on top of the black background of the website)